src/components/GlobalLinterMessages/index.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+import { getCodeLineAnchorID } from '../CodeView/utils';
+import { scrollToSelectedLine as _scrollToSelectedLine } from '../CodeView';
+
+type PublicProps = {
+  className?: string;
```

Comment:
The spec needs a test to make sure that a `className` prop results in an actual class added to the element. Here's an example:

https://github.com/mozilla/addons-code-manager/blob/365b8958bd517c444f4af1008a0eb3beb211cb4b/src/components/FullscreenGrid/ContentShell.spec.tsx#L23